docs/source/contribution.rst

```diff
@@ -174,20 +175,22 @@ In order to run unit tests at the repository root, you first have to build Cytho
   If you want to use ccache for nvcc, please install ccache v3.3 or later.
   You also need to set environment variable ``NVCC='ccache nvcc'``.
 
-Once the Cython modules are built, you can run unit tests simply by running ``nosetests`` command at the repository root::
+Once Cython modules are built, you can run unit tests by running the following command at the repository root::
 
-  $ nosetests
+  $ python -m pytest
 
-It requires CUDA by default.
-In order to run unit tests that do not require CUDA, pass ``--attr='!gpu'`` option to the ``nosetests`` command::
+It requires CUDA and cuDNN by default.
```

Comment:
BTW, I think if a developer does not have CUDA, he cannot develop CuPy...

Reply:
Exactly... fixed.